src/changelog/2025/06/instance-performance-memory.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Instance Performance View
```

Comment:
This `title` and `description` scoped to the whole performance view, but the changelog is actually for the new memory graph. Please re-write them.